doc/data/messages/t/too-many-arguments/bad.py

```diff
@@ -0,0 +1,2 @@
+def foo(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p):  # [too-many-arguments]
+    pass
```

Comment:
```suggestion
def three_d_chess_move(x_white, y_white, z_white, piece_white, x_black, y_black, z_black, piece_black, current_player):  # [too-many-arguments]
    pass
```

Reply:
`black` doesn't like this. I actually struggled to get something to fit on one line but also be over or normal settings value here 😄

Reply:
Let's remove constraints then, it does not have to fit on one line 😄

Reply:
I mean, this is becoming personal preference, but we're not trying to write a "coding style" guideline. Imo, the objective of these snippets is to show why a warning is raised and how to avoid raising it. The `why` behind a message is probably best left to the description or actual guides online.

Reply:
```suggestion
def three_d_chess_move(
    x_white,
    y_white,
    z_white,
    piece_white,
    x_black,
    y_black,
    z_black,
    piece_black,
    x_blue,
    y_blue,
    z_blue,
    piece_blue,
    current_player,
):  # [too-many-arguments]
    pass

```